store/clientpayload.go

```diff
@@ -60,24 +61,24 @@ func (vc WAVersionContainer) String() string {
 	return strings.Join(parts, ".")
 }
 
-// Hash returns the md5 hash of the String representation of this version.
-func (vc WAVersionContainer) Hash() [16]byte {
-	return md5.Sum([]byte(vc.String()))
+// Hash returns the SHA-256 hash of the String representation of this version.
+func (vc WAVersionContainer) Hash() [32]byte {
+	return sha256.Sum256([]byte(vc.String()))
```

Comment:
The hash function can't just be arbitrarily changed, it has to match the one whatsapp uses